langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -39,6 +44,20 @@ public isolated function length(map<any|error> m) returns int =@java:Method {
 
 # Returns an iterator over a map.
 #
+# ```ballerina
+# map<string> vowels = {a: "A", e: "E", i: "I", o: "O", u: "U"};
+#
+# // iterator() returns an iterator object with the `next()` method.
+# var vowelsIter = vowels.iterator();
+#
+# // `next()` can be used to get the next element of the map.
+# record {|string value;|}? nextVal = vowelsIter.next();
+#
+# while nextVal != null {
```

Comment:
This may get removed, but the check should be for `()`. `null` is meant to be used in JSON contexts.